### DISCUSSION TITLE
Public server

### DISCUSSION THREAD

**Discussion by justpassingby0** [200Tigersbloxed/FitbitHRtoWS#13] [Q&A]
Hi,

I had to switch to public server because I couldn't get my data. Now that I can get my HR, how can I use the local HTML file to pull the public server data and use the auto-refresh ? I will only need then to crop that info into OBS and set it as a browser source.

Just missing that part.

Thank you.

**Comment by 200Tigersbloxed** [comment]
This is currently not possible, however I've been wanting to make this a basic GitHub Pages. Maybe later today idk.

But the local file is locked to the local address. It is most likely not compatible with public servers, so I will need to redo it.

Sorry, thanks for the suggestion however!

**Comment by justpassingby0** [comment]
Thanks for the quick reply. What would you suggest I could do to get the data into OBS from the public server ? The local server doesn't seem to work, I might try to give it another try tho. I looked at Pulsoid but the APP doesn't support Fitbit :(

**Comment by justpassingby0** [comment]
I finally got it working locally. and I'm just going to crop the HTML file.
